README.txt

```diff
@@ -77,18 +78,17 @@ lib [This folder contains 3rd party libraries needed in the project]
 		3 - To compile the test, type in the terminal:
 			make clean; make comp; make all
 		4 - To run the test, type in the terminal "./NAME_OF_THE_COMPILED_FILE". For this specific test you need to type:
-			./SUBNET
-		5 - To check the output of the test, open  "/test/data/subnetsubnet_test_output.txt"
+			./SUBNET_TEST
+		5 - To check the output of the test, open  "/test/data/subnet_test_output.txt"
 	2.2. To run receiver and sender tests, the steps are analogous to 2.1
-			
+
 3 - Run the simulator
 	1 - Open the terminal.
 	2 - Set the command prompt in the bin folder. To do so, type in the terminal the path to this folder.
 		Example: cd bin
 	3 - To compile the project, type in the terminal:
 		make clean; make comp; make all
-	4 - To run the simulation, type in the terminal "./NAME_OF_THE_COMPILED_FILE NAME_OF_THE_INPUT_FILE".
-	For this test you need to type:
+	4 - To run the simulation, type in the terminal "./NAME_OF_THE_COMPILED_FILE NAME_OF_THE_INPUT_FILE". For this test you need to type:
 		./ABP ../data/input/input_abp_1.txt
 	5 - To check the output of the simulation, open  "data/output/abp_output.txt"
 	6 - To execute the simulator with different inputs

```

Comment:
in reference to 6.3) I would suggest to consider giving the relative path for renaming the files. Because your current folder location in the terminal is different to where the text files are located.

Reply:
It has been fixed